features/networking/operator.feature

```diff
@@ -121,44 +121,23 @@ Feature: Operator related networking scenarios
     """
 
   # @author bmeng@redhat.com
+  # @author zzhao@redhat.com
```

Comment:
You can take the ownership when updating by replacing bmeng as he no longer works on it.